.github/workflows/pip-release.yml

```diff
@@ -19,61 +19,119 @@ permissions:
 
 jobs:
   linux:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.platform.runner }}
     strategy:
       matrix:
-        target: [x86_64, x86, aarch64]
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+          - runner: ubuntu-latest
+            target: x86
+          - runner: ubuntu-latest
+            target: aarch64
+          - runner: ubuntu-latest
+            target: armv7
```

Comment:
This adds `armv7` as a new target, is that right? If so, we should probably mention this in the changelog/release notes too because right now it's hidden inside a "hot fix" PR.

Reply:
Yes, so I derived this PR to add support for armv7 to close: https://github.com/dora-rs/dora/issues/564 

This make it possible to support Linux version without libc.

I have changed the title to better reflect the fact that we support armv7 using musleabihf

Reply:
> we support armv7 using musleabihf

But `armv7` is specified as part of the `linux` job, not the `musllinux` job for the pip release. So it probably uses the default libc, doesn't it?

Reply:
So `armv7` support is included within `musleabihf`:

https://github.com/dora-rs/dora/blob/11301ab94213eec81134e776a724fcf61189d005/.github/workflows/pip-release.yml#L99-L138

There is both `gnu` <-> `libc` armv7 and `musleabihf` release.

Although it is kind of confusing as in python there is no differenciation between `musl` and `musleabi`

In the case of openharmony, they apparently need `musleabihf`

Reply:
Ah, I missed that, sorry!